Makefile

```diff
@@ -1,3 +1,4 @@
+export GO111MODULE=auto
```

Comment:
what does this do? I see you do it here and in the Dockerfile?

Reply:
a good explanation: https://maelvls.dev/go111module-everywhere/

Reply:
this variable tells `go` to determine if go modules should be enabled. 
the PR updates `go` to `v1.17` that requires modules usage be default.
I want to have prod code unchanged so decided to use the variable to fallback to previous strategy when `go` does not require modules.

It is used in both `Makefile` and `Dockerfile`, because they are independent from each other.
`Dockerfile` is used on `buildkite`.
`Makefile` is used for local development only.